keras/optimizers.py

```diff
@@ -146,11 +146,16 @@ def get_updates(self, params, constraints, loss):
         moments = [K.zeros(shape) for shape in shapes]
         self.weights = [self.iterations] + moments
         for p, g, m in zip(params, grads, moments):
-            v = self.momentum * m - lr * g  # velocity
+            # Apply learning rate multipliers if needed
+            if p in multipliers:
+                lrm = K.variable(multipliers[p])
```

Comment:
Avanti here again (author of the learning rate multipliers implementation on the kundajelab branch that this was based on). Is the K.variable wrapping really necessary? K.variable returns a shared variable, which I understand is only necessary for trainable parameters.